src/main/java/org/candlepin/subscriptions/files/ProductProfile.java

```diff
@@ -92,29 +96,13 @@ public ProductProfile(String name, Set<SubscriptionWatchProduct> products, Granu
         setProducts(products);
     }
 
-    public String getName() {
```

Comment:
Why did these get stripped out?

Reply:
I replaced them with the Lombok annotations.

Reply:
@awood, it looks like a previous PR made the lombok change but these weren't deleted as part of that